README.md

```diff
@@ -1 +1,155 @@
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>HTML, CSS</li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **[Use of linters]**
+- **[Github & Git]**
+- **[README.md file]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [No Live Demo Required](https://google.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Node installed in computer
+```sh
+  npm install npm@latest -g
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd desired-folder
+  git clone https://github.com/Roathy/Hello---Microverse.git
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+Open index.html in browser
+
+### Run tests
+
+To run tests, run the following command:
+
+- No testing implement yet
+
+### Deployment
+
+You can deploy this project using:
+
+GitHub can be used to deploy website. For more information about publishing sources, see "About GitHub pages".
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Julio Eduardo Vergara Martínez**
+
+- GitHub: [@Roathy](https://github.com/Roathy)
+- Twitter: n/a
+- LinkedIn: [LinkedIn](https://linkedin.com/in/julio-vergara-roathy)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Desktop version]**
+- [ ] **[Details section]**
+- [ ] **[Responsiveness ]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Roathy/Hello---Microverse/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project give it a star
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse team.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```

Comment:
- **[OPTIONAL]** I recommend removing this NOTE because is just for us not for the client or user.